CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+* [\#11430](https://github.com/cosmos/cosmos-sdk/pull/11430) Introduce a new `grpc-only` flag, such that when enabled, will start the node in a query-only mode. Note, gRPC MUST be enabled with this flag.
```

Comment:
Can we implicitly enable `gRPC` when this flag is set?

Reply:
I'm not opposed to it :)